feature/session/src/main/res/layout/fragment_bottom_sheet_sessions.xml

```diff
@@ -83,6 +106,42 @@
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toBottomOf="@id/sessions_bottom_sheet_title"
+            app:visibleGone="@{!shouldShowEmptyStateView}"
```

Comment:
maybe needless because of https://github.com/DroidKaigi/conference-app-2019/pull/309/files#diff-f821b14464fd6ed44dda46ce77e85810R45

Reply:
I just noticed, actually this is redundant 😅 . Thank you for pointing. I'll fix. 🔧